test/ctr.bats

```diff
@@ -671,39 +679,43 @@ function teardown() {
 
 @test "ctr execsync std{out,err}" {
 	start_crio
-	run crioctl pod run --config "$TESTDATA"/sandbox_config.json
+	run crictl runs "$TESTDATA"/sandbox_config.json
 	echo "$output"
 	[ "$status" -eq 0 ]
 	pod_id="$output"
-	run crioctl ctr create --config "$TESTDATA"/container_redis.json --pod "$pod_id"
+	run crictl create "$pod_id" "$TESTDATA"/container_redis.json "$TESTDATA"/sandbox_config.json
 	echo "$output"
 	[ "$status" -eq 0 ]
 	ctr_id="$output"
-	run crioctl ctr start --id "$ctr_id"
+	run crictl start "$ctr_id"
 	echo "$output"
 	[ "$status" -eq 0 ]
-	run crioctl ctr execsync --id "$ctr_id" echo hello0 stdout
+	run crictl exec --sync "$ctr_id" echo hello0 stdout
 	echo "$output"
 	[ "$status" -eq 0 ]
-	[[ "$output" == *"$(printf "Stdout:\nhello0 stdout")"* ]]
+	#[[ "$output" == *"$(printf "Stdout:\nhello0 stdout")"* ]]
+	# for now cri-tool exec sync output fromat are different with crio's cli
```

Comment:
nit "fromat" to "format"
Thanks for the explanation.  If this is going to be the long term situation, I'd vote for removing these lines of comments entirely.  Ditto in the other places where you've similar comments.  I think the test as you now have it is fine as is, but @runcom could easily convince me otherwise.